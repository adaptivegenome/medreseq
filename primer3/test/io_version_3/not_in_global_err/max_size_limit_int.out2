../src/primer3_core: PRIMER_INTERNAL_OLIGO_MAX_SIZE exceeds built-in maximum
