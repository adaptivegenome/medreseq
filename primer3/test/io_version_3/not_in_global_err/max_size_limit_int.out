PRIMER_SEQUENCE_ID=max_size_limit_int
SEQUENCE=TAGCTGATGCTAGCTAGCGTGGGCATGCTAGCTAGCTGATGCAGCTAGCTGATCGTAGCTAGCTATCGATCGTAGCTGACTGATCGTAGCTAGCTGATCGATGCTAGCTAGCTAGCTGATCATCTACGTAGCTGACTATCGATGCTACTACTATCGATCGTAGCTAGCTGATCGTAGCTAGCTAGCTGATGCTAC
PRIMER_INTERNAL_OLIGO_MAX_SIZE=45
PRIMER_ERROR=PRIMER_INTERNAL_OLIGO_MAX_SIZE exceeds built-in maximum
=
